data/redirects.yml

```
@@ -700,7 +700,7 @@
   - /go/docker-build-cloud/
 
 # Run Cloud links
-"https://docs.docker.com/":
+"/":
   - /go/run-cloud-eap
```

All other go redirects end with a trailing slash. I don't think it makes a difference, but for consistency's sake, I guess we could have one here as well.

```suggestion
  - /go/run-cloud-eap/
```